### ISSUE TITLE
Would it be possible to switch to the MIT License ? 

### ISSUE THREAD

**Issue by Bavragor** [stephanrauh/AngularFaces#35]
GPL is just so strict... :)
Or Apache 2.0 License maybe.

**Comment by rdebusscher** [comment]
I prefer Apache 2.0.

**Comment by stephanrauh** [comment]
OK! Give me some time to read the licenses first, but most likely I'll add one of the licenses (or even both).

**Comment by tandraschko** [comment]
+1 for apache2 of course :)

**Comment by rdebusscher** [comment]
see http://choosealicense.com/ for example
For each license you also see what is Required, allowed and forbidden.

Don't know if you can combine them (not an expert either)

**Comment by stephanrauh** [comment]
Apache V2 it is.

Dual licensing is not a problem. The licensees can chose themselves which license they want to use. However, Apache V2 is more liberal than GNU GPL in every respect, so keeping GPL is a moot point.

**Comment by Bavragor** [comment]
This would be awesome!

**Comment by stephanrauh** [comment]
I've published RC2 of AngularFaces 2.1 under Apache License V2 (see http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22de.beyondjava%22%20AND%20a%3A%22angularFaces-core%22). However, Maven Central's license checker complains about a license incompatibility. I hope I can fix this.

**Comment by stephanrauh** [comment]
Maven Central's license checker gives me a "go". Starting with AngularFaces 2.1, I put AngularFaces under the Apache License V2. You can download the current release candidate via Maven:
```XML
<dependency>
  <groupId>de.beyondjava</groupId>
  <artifactId>angularFaces-core</artifactId>
  <version>2.1.0-RC3</version>
</dependency>
```

or Gradle:
```groovy
compile 'de.beyondjava:angularFaces-core:2.1.+'
```

or directly from Maven Central: http://search.maven.org/#search%7Cga%7C1%7Cangularfaces.
